CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
Correction: I tried with the initial CMakeLists.txt on OS X: while cmake 3.5.2 finds the PythonLibs, I get the following error when invoking "make":
`    [  0%] Building CXX object src/system/libs/swig/CMakeFiles/_Config.dir/ConfigPYTHON_wrap.cxx.o
    /src/system/libs/swig/ConfigPYTHON_wrap.cxx:155:11: fatal error: 'Python.h'
        file not found
        #include <Python.h>
        1 error generated.
    make[2]: *** [src/system/libs/swig/CMakeFiles/_Config.dir/ConfigPYTHON_wrap.cxx.o] Error 1
    make[1]: *** [src/system/libs/swig/CMakeFiles/_Config.dir/all] Error 2
    make: *** [all] Error 2
`

So should I leave use my code again ?